tests/x11/toolkits/prepare.pm

```diff
@@ -28,11 +29,11 @@ sub run {
             zypper_call('ref');
         }
         elsif (is_sle '>=15') {
-            assert_script_run 'SUSEConnect -p sle-module-development-tools/${VERSION_ID}/${CPU}';
+            add_suseconnect_product('sle-module-development-tools');
         }
         else {
             # for historical reasons we don't register SLE12 systems in openqa by default
-            assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE');
+            register_product();
             assert_script_run 'SUSEConnect -p sle-sdk/${VERSION_ID}/${CPU}';
```

Comment:
@rwx788 shouldn't this line be
```add_suseconnect_product('sle-sdk',${VERSION_ID},${CPU})``

Reply:
Nice catch, yeah we can replace it with just 'add_suseconnect_product('sle-sdk') will update this PR.